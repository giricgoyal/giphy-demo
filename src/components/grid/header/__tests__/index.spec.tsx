import React from 'react'
import { render } from '@testing-library/react'
import '@testing-library/jest-dom'

import GridHeader from '../index'

describe('<GridHeader />', () => {
    let container

    beforeEach(() => {
        ;({ container } = render(
            <GridHeader>
                <div>Name's Bond! James Bond!</div>
            </GridHeader>,
        ))
    })

    test('should match snapshot', () => {
        expect(container).toMatchSnapshot()
    })
})
