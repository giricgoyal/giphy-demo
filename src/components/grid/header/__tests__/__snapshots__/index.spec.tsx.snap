// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GridHeader /> should match snapshot 1`] = `
<div>
  <div
    class="grid__header"
  >
    <div>
      Name's Bond! James Bond!
    </div>
  </div>
</div>
`;
